pkg/cmd/server/start/start_node.go

```diff
@@ -280,8 +274,13 @@ func StartNode(nodeConfig configapi.NodeConfig) error {
 
 	config.EnsureVolumeDir()
 	config.EnsureDocker(docker.NewHelper())
+	plugin, endpointFilter := InitSDNPlugin(config, nodeConfig)
```

Comment:
> no real point to having the fields in NodeConfig if they are only set and used in one function 

it's a staged progression, transforming serialized config into built objects, then starting them